requirements/dev.txt

```diff
@@ -68,19 +69,21 @@ click==7.1.2
     #   code-annotations
     #   edx-lint
     #   pip-tools
-code-annotations==0.10.2
-    # via -r requirements/quality.txt
+code-annotations==1.0.2
```

Comment:
was marked a potentially breaking change b/c it might cause build failures, but didn't against this PR, so /shrug
https://github.com/edx/edx-lint/blob/master/CHANGELOG.rst#400---2021-01-28